monero-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,215 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "dnsLabelPrefix": {
+       "type": "string",
+       "metadata": {
+          "description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+       }
+    },
+    "diskName": {
+       "type": "string",
+       "metadata": {
+          "description": "Enter a name for the OS disk; can be anything, but must be unique."
+       }
+    },
+    "installMethod": {
```

Comment:
This should become a variable. There's only one allowed value for this. No reason to ask the user for this until there's another allowed value.